src/RazorSdk/SourceGenerators/Microsoft.NET.Sdk.Razor.SourceGenerators.csproj

```diff
@@ -12,9 +12,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="$(MicrosoftCodeAnalysisCSharpPackageVersion)" GeneratePathProperty="true" />
-    <PackageReference Include="Microsoft.AspNetCore.Razor.Language" Version="$(MicrosoftAspNetCoreRazorLanguageVersion)" PrivateAssets="all"/>
```

Comment:
What's the motivation for removing this?

Reply:
* We don't need to generate the package path for CodeAnalysis.CSharp (since we're not including it in the SDK)
* CodeAnalysis.Razor references Razor.Language, so an explicit reference to it isn't necessary.
* PrivateAssets="All" is also unnecessary (as we don't grab all the binaries from the build output). It also made it necessary to duplicate the references in the test project.